app/views/events/index.html.erb

```diff
@@ -23,7 +23,8 @@
         <td class="govuk-table__cell" scope="row"><%= event.created_at.strftime('%Y-%m-%d %H:%M:%S.%N') %></td>
         <td class="govuk-table__cell" scope="row"><%= event.aggregate_id %></td>
         <td class="govuk-table__cell" scope="row"><%= event.aggregate_type %></td>
-        <td class="govuk-table__cell" scope="row"><%= event.user_id %></td>
+        <td class="govuk-table__cell" scope="row"><%= @user_list[event.user_id] %>
+        <% unless event.user_id.nil? %>(<%= event.user_id %>)<% end %></td>
```

Comment:
not sure wether we should display user names as well as you added the functionality in `UserInfo.all_users`? Could ask Jakub?